docs/products/flink/howto/real_time_alerting_solution_cli.rst

```diff
@@ -158,7 +163,12 @@ This setup uses a fixed threshold to filter any instances of high CPU load to a
         id2-- high CPU -->id3(Kafka sink);
 
 
-1. Using the Aiven CLI, run the following command to create a Kafka table named ``CPU_IN``:
+1. Using the Aiven CLI, run the following command to create a Kafka table named ``CPU_IN``.
+
+   Replace ``KAFKA_INTEGRATION_ID`` with the ID for your ``demo-kafka`` service integration and replace ``TABLE_SQL`` with the following:
+
+   .. literalinclude:: /code/products/flink/basic_cpu-in_table.md
```

Comment:
In here I see two code blocks one after the other. The first is the sql, the second the `avn command` and I'm not sure the reader will be able to parse it correctly. Since the SQL is "fixed" for each section, should we embed it in the `avn` command itself?

Reply:
This comment applies to all commands where we have the SQL on top and then the `avn` command